esmvaltool/interface_scripts/auxiliary.ncl

```diff
@@ -1028,3 +1088,259 @@ begin
   return(str)
 
 end
+
+; #############################################################################
+undef("get_average")
+function get_average(items: list)
+
+;
+; Arguments
+;    items: list of input_file_info metadata
+;
+; Description
+;    Calculates average of all datasets given by items.
```

Comment:
With respect to which dimension?

Reply:
With respect to the (new) `dataset` dimension, I clarified that in the text